src/App.tsx

```diff
@@ -13,7 +16,13 @@ const client = new ApolloClient({
 const App: React.FC = () => {
     return (
         <ApolloProvider client={client}>
-            <Dashboard />
+            <BrowserRouter>
+                <Switch>
+                    <Route path="/explore" component={DataExplorer} />
```

Comment:
Should we have exact path here aswell?

Reply:
Good idea, might be handy later for nested paths. We work great as a team 🥭🥭😏

Reply:
Thanks man